pages/integrations/inputs/okta_input.rst

```diff
@@ -0,0 +1,42 @@
+.. _okta_input:
+
+*********************
+Okta Log Events Input
+*********************
+
+.. attention:: This is a Graylog Enterprise feature and is only available since Graylog version 3.3. A valid Graylog Enterprise license is required.
+
+
+This input allows Graylog to pull event logs from your Okta domain. For an example of an Okta Log Event object see: https://developer.okta.com/docs/reference/api/system-log/#logevent-object
```

Comment:
> The Okta System Log records system events related to your organization in order to provide an audit trail that can be used to understand platform activity and to diagnose problems.

The above is taken from the referenced page - maybe it would make sense to integrate that too? 

Why someone should install this plugin? What benefit comes out of that? Not in marketing speak - from a technical point of view.

Reply:
Why 

* Utilizing System Log data for real-time monitoring
* provide security alerts